### DISCUSSION TITLE
Web3Auth Firebase support?

### DISCUSSION THREAD

**Discussion by YZhenY** [Web3Auth/Web3Auth#305] [Help]
Hi guys, want to understand Web3Auth integration with firebase authentication is possible currently?

However, as most of our server side assets are protected with firebase auth, so we're looking to extend the firebase provider.

**Comment by shahbaz17** [comment]
Yes, Web3Auth do support Firebase as Federated Identity Provider.

Here's a guide on [Using Firebase with Web3Auth](https://web3auth.io/docs/guides/firebase).

Live Demo: https://demo-firebase.web3auth.io

Also, check out our [Firebase React App](https://github.com/Web3Auth/Web3Auth/tree/master/demo/firebase-react-app) and [Firebase Next.js App](https://github.com/Web3Auth/Web3Auth/tree/master/demo/firebase-next-app) to see how it's implemented.
